integrations/standalone/tests/integration/parts/task.ts

```diff
@@ -58,86 +59,113 @@ export class TaskTester implements PartTest {
   partName() {
     return 'Task';
   }
-  async fill(page: Page) {
-    await FocusCodeEditorUtil.fill(page, page.getByRole('textbox', { name: 'Name' }), this.name);
-    await FocusCodeEditorUtil.fill(page, page.getByLabel('Description'), 'test desc');
-    await FocusCodeEditorUtil.fill(page, page.getByLabel('Category'), 'test cat');
 
-    await SelectUtil.select(page, 'Role from Attr.', 0);
-    await page.getByRole('textbox', { name: 'activator' }).fill('"Teamleader"');
+  async fill(part: Part) {
+    await part.macroInput('Name').fill(this.name);
+    await part.macroArea('Description').fill('test desc');
+    await part.macroInput('Category').fill('test cat');
+
+    const roleChooser = part.responsibleSelect('Responsible');
+    await roleChooser.chooseType('Role from Attr.');
+    await roleChooser.fill('"Teamleader"');
+
+    await part.select('Priority').choose('High');
+
+    const options = part.section('Options');
+    await options.toggle();
+    await options.checkbox('Skip Tasklist').click();
+    await options.input('Delay').fill('delay');
+
+    const expiry = part.section('Expiry');
+    await expiry.toggle();
+    await expiry.scriptInput('Timeout').fill('timeout');
+    await expiry.select('Error').choose(this.error);
+    const responsible = expiry.responsibleSelect('Responsible');
+    await responsible.chooseType('Nobody & delete');
+    await expiry.select('Priority').choose('Low');
+
+    const custom = part.section('Custom fields');
+    await custom.toggle();
+    const table = custom.table();
+    const row = await table.addRow();
+    await row.fill(['cf', 'value']);
+
+    const code = part.section('Code');
+    await code.toggle();
+    await code.scriptArea().fill('code');
+  }
+
+  async assertFill(part: Part) {
+    await part.macroInput('Name').expectValue(this.name);
+    await part.macroArea('Description').expectValue('test desc');
+    await part.macroInput('Category').expectValue('test cat');
+
+    const responsible = part.responsibleSelect('Responsible');
+    await responsible.expectType('Role from Attr.');
+    await responsible.expectValue('"Teamleader"');
 
-    await SelectUtil.select(page, 'High', 1);
+    await part.select('Priority').expectValue(/High/);
 
-    await CollapseUtil.open(page, 'Options');
-    await page.getByRole('checkbox').check();
-    await page.getByLabel('Delay').fill('delay');
+    const options = part.section('Options');
+    await expect(options.checkbox('Skip Tasklist')).toBeChecked();
```

Comment:
I think this checkbox object should also provide a method to check the state, e.g.:
`await options.checkbox('Skip Tasklist').expectChecked(true)`